src/views/v2/rsvp/details/attendance.php

```diff
@@ -26,6 +26,6 @@
 		<?php echo esc_html( $rsvp->qty_sold ); ?>
 	</span>
 	<span class="tribe-tickets__rsvp-attendance-going tribe-common-h7 tribe-common-h--alt tribe-common-b3--min-medium">
-		<?php esc_html_e( 'Going', 'event-tickets' ); ?>
+		<?php echo esc_html_x( 'Going', 'Label below the attendance number.','event-tickets' ); ?>
```

Comment:
:no_entry_sign: No space found after comma in argument list

<sub>hash: e29ea854f396b6ab3bb9e3574723a0a9</sub>